src/cluster/cluster_test.go

```diff
@@ -37,7 +37,7 @@ func (s *ClusterSuite) SetupSuite() {
 	)
 }
 
-func (s *ClusterSuite) GenerateDevices(count int) []*model.Device {
+func GenerateDevices(count int) []*model.Device {
```

Comment:
this is to generate pointerDevices?

Reply:
yep, the pointer isn't too important. It's just what `FromNewDevice` returns.